lib/models/Bike.js

```diff
@@ -0,0 +1,55 @@
+const pool = require('../utils/pool');
+
+module.exports = class Bike {
+  id;
+  brand;
+  model;
+  components;
+
+  constructor(row) {
+    this.id = row.id;
+    this.brand = row.brand;
+    this.model = row.model;
+    this.components = row.components;
+  }
+
+  static async insert({ brand, model, components }) {
+    const { rows } = await pool.query(
+      'INSERT INTO bikes (brand, model, components) VALUES ($1, $2, $3) RETURNING *',
+      [brand, model, components]
+    );
+    if (!rows[0]) {
+      return null;
```

Comment:
We might add these early returns to all of the static methods in the models so that we can avoid errors if no data is returned from a query